src/main/java/seedu/duke/trackables/Task.java

```diff
@@ -0,0 +1,28 @@
+package seedu.duke.trackables;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
```

Comment:
Maybe it would be good to make an enum to label the respective classes (ToDo, Deadline, Event)? This may help in the future when you want to extend your program.